docs/source/OpenAPI.rst

```diff
@@ -0,0 +1,6 @@
+GreedyBear API
+===============
+
+.. openapi:: schema.yaml
+

```

Comment:
about the problem you mentioned before. There are 2 lines here. Remove them

Reply:
Are you referring to the lines 3 and 5?? I removed them but still getting that warning. 
And even in the warning, its referring to the line 44 in the file which doesn't even exist 🤔 .
>/home/docs/checkouts/readthedocs.org/user_builds/greedybear-docs/checkouts/latest/docs/source/OpenAPI.rst:44: WARNING: Definition list ends without a blank line; unexpected unindent.

Reply:
ok. It won't affect docs . I will set the option `fali_on_warning` to `false`  So that builds won't stop because of this warning.